lib/checks/aria/aria-prohibited-attr-evaluate.js

```diff
@@ -26,42 +26,40 @@ import standards from '../../standards';
  * @memberof checks
  * @return {Boolean} True if the element uses any prohibited ARIA attributes. False otherwise.
  */
-function ariaProhibitedAttrEvaluate(node, options, virtualNode) {
-  const prohibited = [];
-
-  const role = getRole(virtualNode);
-  const attrs = virtualNode.attrNames;
-  const prohibitedAttrs = role
-    ? standards.ariaRoles[role].prohibitedAttrs
-    : ['aria-label', 'aria-labelledby'];
-
-  if (!prohibitedAttrs) {
+function ariaProhibitedAttrEvaluate(node, options = {}, virtualNode) {
+  const { elementsAllowedAriaLabel = [] } = options;
+  const attrBlacklist = getAttrBlacklist(virtualNode, elementsAllowedAriaLabel);
```

Comment:
We should avoid the term blacklist.

```suggestion
  const prohibitedAttrs = getProhibitedAttrs(virtualNode, elementsAllowedAriaLabel);
```

Reply:
... fair enough.